dags/process_elife_zip_dag.py

```diff
@@ -165,7 +165,7 @@ def strip_related_article_tags_from_article_xml(**context) -> bytes:
     return etree.tostring(article_xml, xml_declaration=True, encoding='UTF-8')
 
 
-def wrap_article_in_libero_xml_and_send_to_service(**context) -> None:
+def wrap_article_in_libero_xml_and_send_to_service(**context) -> bytes:
```

Comment:
this does not "send to the service" anymore, so rename?

Reply:
updated.